docs/content/any/guides/policies/index.md

```diff
@@ -138,15 +80,13 @@ The logical operators `or` (binary) and `not` (unary) can also
 be used in a rule body, as well as a variety of mathematical,
 matching, and lookup operators.
 
-## The Big Picture
+## Query Evaluation
 
 Before diving any deeper into the details of rules and matching,
 let's take a moment to put them in context. Rules are loaded into
 a **knowledge base**, a kind of specialized in-memory database that
 supports **queries** by pattern matching and logical inference.
 
-![Oso Architecture](guides/policies/arch.svg)
```

Comment:
we should redo a similar diagram w/ `authorize()` instead of `is_allowed()`

Reply:
follow up